source/frontend/src/features/mapSideBar/property/tabs/propertyDetails/update/validation.ts

```diff
@@ -35,4 +37,17 @@ export const UpdatePropertyDetailsYupSchema = Yup.object().shape({
     postal: Yup.string().max(20, 'Postal must be at most 20 characters'),
   }),
   generalLocation: Yup.string().max(2000, 'General location must be less than 2000 characters'),
+  historicalNumbers: Yup.array().of(
+    Yup.object().shape({
+      historicalNumber: Yup.string(),
```

Comment:
this should have a character limit of 500 characters.

Reply:
Will add